[REF] Update the example query results (#186)
neurobagel/documentation: docs/query_tool.md

Note that this ID is not guaranteed to be persistent across versions of a graph/across graphs, but will always be identical across a pair of query tool result files. 
_This column can be used as the key to join the dataset-level and participant-level results TSVs for a given query result, if needed._
- `SubjectID`: subject label

> - `SessionID`: the label of the session (can be empty if there is no session)

Reviewer: ```suggestion
- `SessionID`: the label of the session
```

I don't think `SessionID` can actually be empty, since all phenotypic/imaging data for a subject will be created under either a phenotypic or imaging session (if a session doesn't exist in the raw data, we just create one in the CLI).

- `SessionType`: either `ImagingSession` or `PhenotypicSession`
- `Age`: subject age, if available
- `Sex`: subject sex, if available
- `Diagnosis`: list of diagnoses of subject, if available
- `Assessment` : list of assessments completed by subject, if available

> - `SessionFilePath`: the path of the session directory relative to the dataset root (for datasets available through DataLad) or root of the filesystem where the dataset is stored

Reviewer: ```suggestion
- `SessionFilePath`: the path of the session directory (for imaging sessions) either relative to the dataset root (for datasets available through DataLad) or relative to the root of the filesystem where the dataset is stored
```

> - `NumPhenotypicSessions`: (aggregate variable) total number of available **phenotypic** sessions for subject. 
> - `NumImagingSessions`: (aggregate variable) total number of available **imaging** sessions for subject. 

Reviewer: ```suggestion
- `NumPhenotypicSessions`: (aggregate variable) total number of **phenotypic** sessions that match the query for a subject
- `NumImagingSessions`: (aggregate variable) total number of **imaging** sessions that match the query for a subject
```
